src/rapidata/rapidata_client/rapidata_client.py

```diff
@@ -85,12 +91,38 @@ def get_order(self, order_id: str) -> RapidataOrder:
         # TODO: check the pipeline for the dataset id - not really necessary atm
         # order = self.openapi_service.order_api.order_get_by_id_get(order_id)
         # pipeline = self.openapi_service..pipeline_get_by_id_get(order.pipeline_id)
+        try:
+            order = self.openapi_service.order_api.order_get_by_id_get(order_id)
+        except Exception:
+            raise ValueError(f"Order with ID {order_id} not found.")
```

Comment:
Nit: Should we still print the original error message as well to make debugging easier in case this fails?